guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc

```diff
@@ -1,6 +1,6 @@
-[[upgrading_clients]]
+[[upgrading_content_hosts]]
```

Comment:
renaming this topic and all the surrounding changes seem rather out of scope in this PR and make reading the diff harder :(